mesonbuild/modules/rust.py

```diff
@@ -245,6 +246,11 @@ def bindgen(self, state: ModuleState, args: T.List, kwargs: FuncBindgen) -> Modu
 
         return ModuleReturnValue([target], [target])
 
+    def bindgen_version(self, state: ModuleState, args: T.List, kwargs: FuncBindgen) -> str:
```

Comment:
```suggestion
   @noPosargs
   @noKwargs
    def bindgen_version(self, state: ModuleState, args: T.List, kwargs: T.Dict[str, T.Any]) -> str:
```

With the appropriate imports, and I think this is good to go